packages/api/src/routers/svc/newsletters.ts

```diff
@@ -128,6 +130,16 @@ export function newsletterServiceRouter() {
           return res.status(500).send('Error saving url from email')
         }
       } else {
+        // do not subscribe if subscription already exists and is unsubscribed
+        const existingSubscription = await getSubscriptionByNameAndUserId(
+          data.author,
+          newsletterEmail.user.id
+        )
+        if (existingSubscription?.status === SubscriptionStatus.Unsubscribed) {
```

Comment:
nice, so these wont get put into recents, right?

Reply:
@jacksonh i choose to put these into recent emails so that a user could still subscribe a `Unsubscribed` newsletter by marking the email as an article

let me know if there is a better way to handle it

Reply:
Yeah I would say still going to recents makes sense, maybe it could have a status like `unsubscribed` or something in the future.

Reply:
yeah, that is a good idea!